internal/cli/universal_login.go

```diff
@@ -353,7 +353,7 @@ func (cli *cli) customTemplateEditorPromptWithPreview(cmd *cobra.Command, body *
 	defer cancel()
 
 	onInfo := func() {
-		cli.renderer.Infof("%s once you close the editor, you'll be prompted to save your changes. To cancel, CTRL+C.", ansi.Faint("Hint:"))
+		cli.renderer.Infof("%s once you close the editor, you'll be prompted to save your changes. To cancel, press CTRL+C.", ansi.Faint("Hint:"))
```

Comment:
Some hints are sentence-cased following the "Hint:", other begin with a lowercase letter. Should we stick to a single style?

Reply:
Good catch, we decided to upper-case them all 👍